libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.component.ts

```diff
@@ -164,7 +164,7 @@ export class SkyTileDashboardComponent implements AfterViewInit, OnDestroy {
   }
 
   #checkReady(): void {
-    if (this.#viewReady && this.config && this.columns && this.singleColumn) {
```

Comment:
This is the line that fixed the issue. The rest of the changes (aside from the additional unit test) are minor cleanup changes for issues I noticed while in the code.